iothub_client/tests/common_longhaul/iothub_client_common_longhaul.c

```diff
@@ -1505,7 +1549,7 @@ static int invoke_device_method(const void* context)
             device_method_info.method_id = method_id;
             device_method_info.time_invoked = time(NULL);
 
-            for (int i = 0; i < 3; i++)
+            for (int i = 1; i <= 3; i++)
```

Comment:
Nit - you cleaned up some other magic constants in this PR, `3` is another candidate.